modules/ding_library/ding_library.features.field_instance.inc

```diff
@@ -81,8 +81,8 @@ function ding_library_field_default_field_instances() {
       'module' => 'addressfield',
       'settings' => array(
         'available_countries' => array(
-          'DK' => 'DK',
           'DE' => 'DE',
+          'DK' => 'DK',
```

Comment:
This change does lot seem relevant to the issue. Will it have any functional changes to the interface?

For example: We do not want German addresses to be the default because the ordering was changed in the configuration.